src/pages/ReplaceServer/ReplaceController.js

```diff
@@ -38,6 +38,13 @@ class ReplaceController extends BaseUpdateWizardPage {
 
   setNextButtonDisabled = () => this.state.overallStatus != STATUS.COMPLETE;
 
+  getHostname = (internalModel, serverId) => {
+    const matches = internalModel.internal.servers.filter(s => s.id == serverId).map(s => s.hostname);
```

Comment:
Based on my experience of adding compute,   if you need set --limit,  use day0-cp1-c1-m1 which is in server['ardana_ansible_host']. server['hostname'] gives  something like day0-cp1-c1-m1-mgmt. I never run osconfig-run.yml ...you might want to run it to see what it takes for limit.

Reply:
The operations guide says it is supposed to be server['hostname'], the one with the -mgmt extension.

Reply:
if you checked, that sounds good. It is kind of messy that ansible playbook limit sometimes takes hostname with -mgmt and sometimes without.